src/components/PopupWithForm.js

```diff
@@ -29,13 +41,18 @@ class PopupWithForm extends Popup {
   // Submits the form by calling the handleFormSubmit function with the input values
   _submitForm = () => {
     const inputValues = this._getInputValues();
+    console.log(inputValues);
     this._handleFormSubmit(inputValues);
   };
 
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`